oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -267,6 +265,37 @@ discussion category
 <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_ or in the
 ``#core-contributors`` room on Slack.
 
+Removing Core Contributors
+--------------------------
+
+Occasionally, it may become necessary to remove an individual from the Core
+Contributor program. Some reasons this may happen include:
+
+* A CC leaves the project or changes employment and does not wish to continue
+  participating
+* A CC does not have the time or interest to continue in the role
+* A CC is not fulfilling the responsibilities of the role
+* A CC is not upholding the standards of the community
+
+Trigger events for removing a CC would be:
+
+* A CC informs the Program Administrators that they can no longer continue in
+  the role
+* A CC has been inactive and unreachable for more than 6 months
+* A CC hasn't met project quality standards consistently despite feedback
+* A CC has consistently shown poor judgement, such as merging PRs without
+  approval
```

Comment:
CCs can approve PRs and merge them without outside approval, so maybe reword this?

```suggestion
* A CC has consistently shown poor judgement, such as merging PRs without due consideration
```

Reply:
Ah, great catch. My intention was something along the lines of "merging PRs without addressing outstanding concerns" - your wording works for me.